src/test/obj_ctl_arenas/obj_ctl_arenas.c

```diff
@@ -159,7 +159,22 @@ main(int argc, char *argv[])
 
 		os_mutex_destroy(&lock);
 		os_cond_destroy(&cond);
+	} else if (t == 'c') {
+		unsigned narenas_b = 0;
+		unsigned narenas_a = 0;
+		unsigned narenas_n = 4;
+
+		ret = pmemobj_ctl_get(pop, "heap.narenas", &narenas_b);
+		UT_ASSERTeq(ret, 0);
 
+		for (unsigned i = 0; i < narenas_n; i++) {
+			ret = pmemobj_ctl_exec(pop, "heap.arena.create", NULL);
+			UT_ASSERTeq(ret, 0);
+		}
+
+		ret = pmemobj_ctl_get(pop, "heap.narenas", &narenas_a);
+		UT_ASSERTeq(ret, 0);
+		UT_ASSERTeq(narenas_b + narenas_n, narenas_a);
```

Comment:
You would need to check if you can allocate from the newly created arena (you can't right now :P)